packages/core/package.json

```diff
@@ -49,6 +49,7 @@
     "deepmerge": "^4.2.2",
     "dotenv": "^16.0.0",
     "etag": "^1.8.1",
+    "ethers": "^6.6.7",
```

Comment:
📝  This will be in connector package and used through `generateNonce` and `verifyMessage` functions provided by the connector, not core